product_docs/docs/tde/15/enabling_tde_epas.mdx

```diff
@@ -28,13 +28,15 @@ and use `pg_upgrade` to transfer data from the existing source cluster to the ne
 
 ## Worked example
 
+This example enables Transparent Data Encryption on an EDB Postgres Advanced Server version 16 running on an Ubuntu 22.04 machine. The location of the BIN and CONFIG directories differs depending on your operating system and Postgres versions.
+
```

Comment:
instead of just saying "EDB Postgres Advanced Server" -  Could  we say 
"This example enables Transparent Data Encryption on an EDB Postgres Advanced Server 16 and EDB Postgres Extended Server 16 running on ..... " 

 as we are trying to cover both the servers.